invenio_app_rdm/records_ui/views/filters.py

```diff
@@ -96,7 +96,7 @@ def doi_identifier(identifiers):
     for identifier in identifiers:
```

Comment:
moderate: I think there was a missunderstanding here... if you use the `get` there is no need to loop on top of it :)